tests/app/celery/test_broadcast_message_tasks.py

```diff
@@ -331,6 +374,11 @@ def test_send_broadcast_provider_message_errors(mocker, sample_service):
         sent=event.sent_at_as_cap_datetime_string,
         expires=event.transmitted_finishes_at_as_cap_datetime_string,
     )
+    mock_retry.assert_called_once_with(
+        countdown=1,
```

Comment:
Optional: It took me a second to work out that `countdown=1` is because this is the first retry so it's 2^0=1 and so this is the assertion that is checking it's exponential (because otherwise it would be countdown 0 or something else). Your other option could be to mock `get_retry_duration` and assert the countdown is the return value of the mock

Reply:
properly patched the retry count and parametrized